src/OCore/OCore.Entities.Data.Http/Mapping.cs

```diff
@@ -38,7 +31,8 @@ private static IEnumerable<Type> GetAllTypesThatImplementInterface<T>()
             return AppDomain.CurrentDomain
             .GetAssemblies()
             .SelectMany(x => x.GetTypes())
-            .Where(type => !type.IsInterface && (type.GetInterfaces().Contains(typeof(T)) || typeof(T).IsAssignableFrom(type)));
+            .Where(type => type.IsInterface && !string.IsNullOrEmpty(type.Namespace) &&
+                   !type.Namespace.StartsWith("OrleansCodeGen") && type.GetInterfaces().Contains(typeof(T)));
```

Comment:
@COCPORN We need to talk about how we discover and map types. This is a bit of a hacky way I think, especially when i'm avoiding the OrleansCodeGen namespace - Ill see what other options are available.

Reply:
![image](https://user-images.githubusercontent.com/2087337/176876325-aed72802-cb69-44b4-b617-17f9880ccb3f.png)

Reply:
![image](https://user-images.githubusercontent.com/2087337/176876470-d7431f0a-50c7-427f-85c9-c20cee4c61f1.png)

Reply:
I will exclude based on `GeneratedCode` attribute